docs/getting-started/installing/troubleshooting.md

```diff
@@ -0,0 +1,156 @@
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/2.3.0/net/http.rb:878:in `connect'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/2.3.0/net/http.rb:852:in `start'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/2.3.0/net/http.rb:1398:in `request'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/etcd-0.3.0/lib/etcd/client.rb:111:in `api_execute'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /app/app/etcd_client.rb:27:in `version'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /app/app/etcd_client.rb:22:in `initialize'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /app/config.ru:11:in `new'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /app/config.ru:11:in `block in <main'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/rack-1.6.4/lib/rack/builder.rb:55:in `instance_eval'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/rack-1.6.4/lib/rack/builder.rb:55:in `initialize'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /app/config.ru:1:in `new'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /app/config.ru:1:in `<main'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/rack/adapter/loader.rb:33:in `eval'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/rack/adapter/loader.rb:33:in `load'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/rack/adapter/loader.rb:42:in `for'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/thin/controllers/controller.rb:170:in `load_adapter'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/thin/controllers/controller.rb:74:in `start'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/thin/runner.rb:200:in `run_command'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/thin/runner.rb:156:in `run!'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/bin/thin:6:in `<top (required)'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/bin/thin:23:in `load'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/bin/thin:23:in `<top (required)'
+```
+
+
+## Possible causes
+
+### Kontena agent peer interface is incorrect
+
+Kontena nodes comunicates via a configured peer network which defaults to __eth1__ and fallback to __eth0__. 
+
+If your distribution or environment is using a different interface name, comunication is compromised.
```

Comment:
"comunication" --> communication.

Should be opened up a bit what communication is compromised. It's the overlay network connections between nodes.